src/Adapter/Order/AbstractOrderHandler.php

```diff
@@ -325,7 +326,10 @@ protected function getProductRegularPrice(
             false,
             $order->id_customer, // We still use the customer ID in case this customer has some special prices
             null, // But we keep the cart null as we don't want this order overridden price
-            $order->{Configuration::get('PS_TAX_ADDRESS_TYPE', null, null, $order->id_shop)}
+            $order->{Configuration::get('PS_TAX_ADDRESS_TYPE', null, null, $order->id_shop)},
+            $specificPriceOutput,
```

Comment:
Since you don't use the output you can use `null` here

Reply:
the parameter is passed by reference, I can only use a variable, even if it's null